packages/core/src/feed.ts

```diff
@@ -0,0 +1,60 @@
+import { Subject, Observable, map } from 'rxjs'
+import type { StreamMetadata, StreamState } from '@ceramicnetwork/common'
+import { CommitID } from '@ceramicnetwork/streamid'
+import type { AnchorStatus } from '@ceramicnetwork/common'
+
+export type Document = {
+  id: CommitID
```

Comment:
Could we make it more explicit the ID exposed here is a `CommitID` please?
I think it could be confusing otherwise for consumers that are used to having the `StreamID` provided.
```suggestion
  commitID: CommitID
```

Reply:
Ready!